t/op/numconvert.t

```diff
@@ -159,8 +159,10 @@ for my $num_chain (1..$max_chain) {
 	      if ($curop < 5) {
 		if ($curop < 3) {
 		  if ($curop == 0) {
+            no warnings 'imprecision';
```

Comment:
something is wrong with the indentation, but not a blocker

Reply:
> something is wrong with the indentation, but not a blocker

I'll have to tabs -> spaces for the entire `for` loop.  Doing so now.

Reply:
> > something is wrong with the indentation, but not a blocker
> 
> I'll have to tabs -> spaces for the entire `for` loop. Doing so now.

Done.